docs/how-to-guides/create-multi-node-cluster.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Create a Multi-Node Cluster with k3d
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Rancher Desktop provides a `single cluster with single node` setup, which is adequate for most local development scenarios. However, there are use cases where, the ability to create a multi node cluster or spin up multiple clusters with flexibilty to switch between clusters is required. Eventhough Rancher Desktop doesn't have in-built multi node/cluster functionality, you can use [k3d](https://k3d.io) with Rancher Desktop to accomplish the same. k3d is a lightweight wrapper to run k3s (SUSE Rancher’s minimal Kubernetes distribution, which is used by Rancher Desktop as well) in docker. k3d makes it very easy to create single- and multi-node k3s clusters in docker, e.g. for local development on Kubernetes.
+
+### Steps to spin up a multi-node cluster
+
+1. Make sure `dockerd(moby)` is selected as the Container Runtime in the `Kubernetes Settings` page.
+
+2. Install k3d.
+
+<Tabs groupId="installation-approach">
+  <TabItem value="wget" default>
+
+```
+wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
+```
+
+  </TabItem>
+  <TabItem value="curl">
+
+```
+curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
+```
+
+  </TabItem>
+</Tabs>
+
+3. Run `k3d cluster create` command to spin up multi node clusters. For example:
+
+```
+k3d cluster create two-node-cluster --agents 2
+k3d cluster create three-node-cluster --agents 3
+```
```

Comment:
Should we point people to k3s docs for more information on using it?

Reply:
Added links to k3s and k3d docs